package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "canopy-webpack-config",
+  "version": "1.0.0",
+  "description": "Some defaults for webpack configs at canopy",
+  "main": "src/canopy-webpack-config.js",
+  "repository": "git@github.com:CanopyTax/canopy-webpack-config.git",
+  "author": "Joel Denning <joel.denning@canopytax.com>",
+  "license": "Apache-2.0",
+  "devDependencies": {},
+  "dependencies": {
+    "clean-webpack-plugin": "^0.1.19",
```

Comment:
normally clean-webpack-plugin is a devDependency because you only use it locally to build stuff.

But in our case, it needs to be a `dependency` because that way it will be installed when people install canopy-webpack-config and be available for use.